Gemfile

```diff
@@ -57,8 +57,11 @@ gem 'geocoder'
 gem 'underscore-rails'
 gem 'gmaps4rails'
 gem 'searchbing'
+gem 'google-search'
+gem "google_custom_search_api"
 gem 'kaminari'
 gem 'kaminari-bootstrap'
+gem 'sidekiq'
```

Comment:
Delayed Job is still there, please remove it and ensure things are working with Sidekiq. See https://github.com/CodeCoreYVR/vancouvertechhub/pull/4/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R48